templates/corporate/attribution.html

```diff
@@ -2,9 +2,12 @@
 {% set entrypoint = "landing-page" %}
 
 {% block title %}
-<title>Zulip: The best group chat for your business.</title>
+<title>Zulip attributions</title>
 {% endblock %}
 
+{% set OPEN_GRAPH_TITLE = 'Zulip attributions' %}
```

Comment:
I think "Zulip website attributions" would be more appropriate; these are only for the website, not the app.